### PR TITLE
DEV-4416: Integrate Siklu unit tests

### PR DESCRIPTION
**Pull request by eduardpo**
## Description
Support for Siklu pyangbind/tests,
Unit tests framework used for testing pyangbind functionality.

## Tests
tests/serialise/xml-deserialise/run_siklu.py
pybindIETFXMLEncoder() & pybindIETFXMLDecoder() validation using Siklu YANG input.

## References
JIRA: DEV-4416

### CODE REVIEW

**Comment by eduardpo** [comment]
@shmuelhazan

**Comment by eduardpo** [comment]
> @shmuelhazan

File: tests/serialise/xml-deserialise/run_siklu.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env python
 
-import os.path
+import os
```
Comment (shmuelhazan) [inline tests/serialise/xml-deserialise/run_siklu.py:3]: There are some code comments in these files. Do we need them?
Also, missing explanation of what this file is doing.

**Review by shmuelhazan — CHANGES_REQUESTED** [review]

File: tests/serialise/xml-deserialise/run_siklu.py (hunk 1 above)
Comment (eduardpo) [reply tests/serialise/xml-deserialise/run_siklu.py:3]: actually not, i've left them only for reference...

Comment (eduardpo) [reply tests/serialise/xml-deserialise/run_siklu.py:3]: i'll add the description

Comment (shmuelhazan) [reply tests/serialise/xml-deserialise/run_siklu.py:3]: I think that as a rule of thumb, it is always better to remove code that is not running.

**Comment by eduardpo** [comment]
@shmuelhazan

**Review by shmuelhazan — APPROVED** [review]

**Pull request merged**